### PR TITLE
Sirigen

### PR DESCRIPTION
**Pull request by sirireddy24**

### CODE REVIEW

File: generators/utilityscripts/main/deploy/deploy.ps1 [hunk 1]
```diff
@@ -1,10 +1,21 @@
-function Uninstall() {
+function Uninstall()
+{
     Write-Host "Uninstalling App as installation failed... Please try installation again."
     Invoke-Expression "& $PSScriptRoot\uninstall.ps1"
     Exit
 }
-
-$AppPath = "$PSScriptRoot\<%= appPackage %>"
+$path="./vscode-config.json"
```
Comment (anantshankar17) [inline generators/utilityscripts/main/deploy/deploy.ps1]: this name should be  <%= VSCodeConfigName %> and should be updated by the generator at the time of deploy script creation.

File: generators/utilityscripts/main/deploy/deploy.sh [hunk 2]
```diff
@@ -6,6 +6,14 @@ uninstall () {
 }
 
 cd `dirname $0`
+path="./vscode-config.json"
```
Comment (anantshankar17) [inline generators/utilityscripts/main/deploy/deploy.sh]: Same as above.

File: package.json [hunk 3]
```diff
@@ -17,10 +17,13 @@
     "yeoman-generator"
   ],
   "dependencies": {
+    "chalk": "^1.1.3",
+    "find": "^0.3.0",
```
Comment (anantshankar17) [inline package.json]: remove those dependencies which are used.

File: package-lock.json [hunk 4]
```diff
@@ -0,0 +1,5439 @@
+{
+  "name": "generator-azuresfcsharp",
```
Comment (anantshankar17) [inline package-lock.json]: This seems like a generated file, is it required to be checked in ? 
Can you remove if not required ?

**Comment by sirireddy24** [comment]
I didn’t add that. It automatically got updated for some reason. Should I remove it?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Pull request closed without merging**